.github/workflows/tiobe-scan.yaml

```diff
@@ -0,0 +1,103 @@
+name: TIOBE scan
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1" # Runs every Monday at 00:00 UTC.
+
+jobs:
+  build:
+    runs-on: [self-hosted, linux, amd64, tiobe, jammy]
+    steps:
+    - name: Checkout the project
+      uses: actions/checkout@v4
+
+    - name: Setup Go
+      uses: actions/setup-go@v4
+      with:
+        go-version-file: "go.mod"
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update -y && sudo apt-get install -y gcc git-core gnupg build-essential
+        go install honnef.co/go/tools/cmd/staticcheck@v0.6.1
+        go install github.com/t-yuki/gocover-cobertura@latest
+    - uses: actions/checkout@v4
+    - uses: actions/setup-go@v5
+      with:
+        go-version-file: "go.mod"
+        cache: true
+    - uses: hashicorp/setup-terraform@v3
+      with:
+        terraform_version: "1.11.*"
+        terraform_wrapper: false
+    - name: Setup operator environment
+      uses: charmed-kubernetes/actions-operator@main
+      with:
+        provider: "lxd"
+        channel: "6"
+        juju-channel: "3/stable"
+    - name: In case of LXD setup also microk8s
+      run: |
+        sudo snap install microk8s --channel=1.32-strict/stable
+        sudo mkdir -p /var/snap/microk8s/current/args/certs.d/docker.io
+        cat <<EOF | sudo tee /var/snap/microk8s/current/args/certs.d/docker.io/hosts.toml
+        server = "$DOCKERHUB_MIRROR"
+        [host."$DOCKERHUB_MIRROR"]
+        capabilities = ["pull", "resolve"]
+        EOF
+        sudo microk8s stop
+        sudo microk8s start
+        sudo usermod -a -G snap_microk8s $USER
+        sudo chown -R $USER ~/.kube
+        sudo microk8s.enable dns hostpath-storage
+        sudo -g snap_microk8s -E microk8s status --wait-ready --timeout=600
+    - name: Create additional networks when testing with LXD
+      run: |
+        sudo lxc network create management-br ipv4.address=10.150.40.1/24 ipv4.nat=true ipv6.address=none ipv6.nat=false
+        sudo lxc network create public-br ipv4.address=10.170.80.1/24 ipv4.nat=true ipv6.address=none ipv6.nat=false
+    - name: "Set additional environment for LXD"
+      # language=bash
+      run: |
+       sudo microk8s.config > /home/$USER/microk8s-config.yaml
+    - name: "Set environment to configure provider"
```

Comment:
This will be a pain to remember to update if we update the test workflow.
In the future it would be good to rework the CI a bit to avoid this issue.